enricher/standard/src/test/java/io/fabric8/maven/enricher/standard/ConfigMapEnricherTest.java

```diff
@@ -27,55 +27,64 @@
 import mockit.Expectations;
 import mockit.Mocked;
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
 
 import java.io.IOException;
-import java.net.URISyntaxException;
 import java.nio.file.Files;
+import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
nit, avoid wildcard imports.

Reply:
Of course! I have to fix the code formatting template in IDEA on my home laptop. Fixed now.